rllib/algorithms/dreamer/dreamer.py

```diff
@@ -409,8 +440,6 @@ def training_step(self) -> ResultDict:
                 gif = policy_fetches["log_gif"]
                 policy_fetches["log_gif"] = self._postprocess_gif(gif)
 
-        self.local_replay_buffer.add(batch)
```

Comment:
I moved this up to the sampling steps of this training step, for the sake of keeping this training step clean. Please step forward if you have thoughts about algorithmic correctness - not an expert on dreamer.